io.github.sameboy.SameBoy.yml

```
@@ -0,0 +1,52 @@
+app-id: io.github.sameboy.SameBoy
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm13
+build-options:
+  append-path: /usr/lib/sdk/llvm13/bin
+  prepend-ld-library-path: /usr/lib/sdk/llvm13/lib
+command: sameboy
+rename-desktop-file: sameboy.desktop
+rename-icon-file: sameboy
+finish-args:
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=ipc
+  - --device=dri
+  - --filesystem=home
+
+modules:
+  - name: rgbds
+    buildsystem: cmake-ninja
+    builddir: true
+    sources:
+      - type: git
+        url: https://github.com/gbdev/rgbds.git
+        tag: v0.5.2
+        commit: 9d993d84e85eeb8dd304e48463d89865795e97ed
+
+  - name: sameboy
+    buildsystem: autotools
+    no-autogen: true
+    build-options:
+      arch:
+        aarch64:
+          cflags: -Wno-unused-command-line-argument
```

```suggestion
          cflags: -Wno-error
```

Just not have warning cause error. Unless you are going to actually fixt them